packages/frameworks/vue/src/dialog/stories/dialog.story.vue

```diff
@@ -9,7 +9,7 @@ const open = ref(false)
 <template>
   <Story title="Dialog">
     <Variant title="Basic">
-      <Dialog.Root>
+      <Dialog.Root default-open>
```

Comment:
```suggestion
      <Dialog.Root>
```
To sync with other stories